### PR TITLE
Suppress unmap errors at undo_ftplugin

### PR DESCRIPTION
**Pull request by grauwoelfchen**
This pull request suppresses errors relate to unmappings at unloading of this plugin.

The `nmap`/`vmap` both are currently set only if those mappings aren't set yet by others (with chesk of `maparg(...)` at [L44](https://github.com/wlangstroth/vim-racket/blob/3948cc5f100b1d1f7d7e4c302e99527c1858e229/ftplugin/racket.vim#L44-L46) and [L63](https://github.com/wlangstroth/vim-racket/blob/3948cc5f100b1d1f7d7e4c302e99527c1858e229/ftplugin/racket.vim#L63-L65)). So, if these buffer local mappings are not set (I had another mapping for `K`), then errors like below occur when `b:undo_ftplugin` is evaluated, since they don't exist.

```txt
Error detected while processing BufRead Autocommands for "*.rkt"..function <SNR>55_RacketDetectHashLang[3]..FileType Autocommands 
for "*"..function <SNR>20_LoadFTPlugin:
line    3:
E31: No such mapping
```

I suppose these errors would be harmless (because buffer local mappings does not exist), but a bit annoying, so I've simply added `silent!` at here 😄

### CODE REVIEW

**Comment by grauwoelfchen** [comment]
I found this during an investigation of another issue at https://github.com/purescript-contrib/purescript-vim/pull/91, because this unloading part seems to be unexpectedly? evaluated due to that when I just opened a Racket file.

**Comment by benknoble** [comment]
Something similar is already achieved in https://github.com/benknoble/vim-racket/blob/1c02b65f30d01eb05049e0a3f1d7ddb84658457c/ftplugin/racket.vim#L64 which ships with latest Vim. I don't remember why `execute` is necessary but I'm fairly confident it is (possibly to do with separating from the commands that come after).

**Comment by grauwoelfchen** [comment]
That's nice!

Speaking of `execute`, I also had an experiment unmapping them conditionally with `if` like below. Maybe that's a bit wordy, but it somehow worked even without `silent!`. This version didn't work without using those `exucute`s 🤔 

```viml
let b:undo_ftplugin =
      \  "setl iskeyword< lispwords< lisp< comments< formatoptions<"
      \. "| setl makeprg< commentstring<"
      \. "| if maparg('K', 'n') ==# '<Plug>RacketDoc' \| execute 'nunmap <buffer> K' \| endif"
      \. "| if maparg('K', 'v') ==# '<Plug>RacketDoc' \| execute 'vunmap <buffer> K' \| endif"
```

Anyway, I agree with that. That would fairly be better to have it in a string for clear separation.
So I'll add `execute`s here too. Thanks 🙋🏽‍♀️
